openpype/plugins/publish/validate_containers.py

```diff
@@ -24,5 +30,9 @@ class ValidateContainers(pyblish.api.ContextPlugin):
     actions = [ShowInventory]
 
     def process(self, context):
+        if not self.is_active(context.data):
+            return
```

Comment:
Shouldn't this be Pyblish's responsibility? Or why do we need to do this manually?